kalite/distributed/static/css/distributed/sidebar.css

```diff
@@ -134,7 +131,20 @@ div.fade {
     color: #CCC;
 }
 
-.back-to-parent {
+.sidebar-back {
+    height: 100%;
+    width: 45px;
+    padding-top: 100px;
+    text-align: center;
+    color: #fff;
+    background-color: #467471;
+    font-size: 24px;
+    border-right: 1px solid rgba(0, 0, 0, 0.3);
+    position: absolute;
+    z-index: 10000;
+}
+
+/*.back-to-parent {
```

Comment:
Is it safe to remove these styles? Did you remove the elements that were styled by this? Better to delete instead of cluttering up the file with commented-out code.